scripts/cache_filter_smudge.sh

```diff
@@ -0,0 +1,5 @@
+temp_out=$(mktemp)
+cat > $temp_out.ipynb
```

Comment:
Here you're doing something odd: first line creates `temp_out` file; second line creates `temp_out.ipynb`. Better skip the extension.